apps/delivery-options/src/composables/usePickupLocation.ts

```diff
@@ -11,9 +11,18 @@ interface UsePickupLocation {
 
 export const usePickupLocation = (locationCode: MaybeRef<string | undefined>): UsePickupLocation => {
   const {locations} = useResolvedPickupLocations();
+  const currentLocationCode = ref(get(locationCode));
+
+  watch(
+    () => get(locationCode),
+    (newCode) => {
+      currentLocationCode.value = newCode;
+    },
+    {immediate: true},
+  );
 
   const pickupLocation = computed(() => {
-    return toValue(locations).find((location) => location.locationCode === get(locationCode));
+    return toValue(locations).find((location) => location.locationCode === currentLocationCode.value);
```

Comment:
`toValue` and `get` are mostly the same, and are now used interchangeably here, along with direct `.value` access.

Prefer to use `toValue` where possible, as that's in Vue core.